Sming/Components/rboot/include/Network/RbootHttpUpdater.h

```diff
@@ -18,32 +18,101 @@
 #include <Network/HttpClient.h>
 #include "../Data/Stream/RbootOutputStream.h"
 
-#define NO_ROM_SWITCH 0xff
+/**
+ * @brief Magic value for ROM slot indicating slot won't change after successful OTA
+ */
+constexpr uint8_t NO_ROM_SWITCH{0xff};
 
 class RbootHttpUpdater;
 
-typedef Delegate<void(RbootHttpUpdater& client, bool result)> OtaUpdateDelegate;
-
-struct RbootHttpUpdaterItem {
```

Comment:
Since `RbootHttpUpdaterItem` this is gone and replace with `Item`  we should add it to the upgrade document with hints what needs to be done.

Reply:
AFAIK it's an internal variable, but as it's returned  from `getItem` I'll add it back as a deprecated alias.